examples/adminapi.py

```diff
@@ -890,8 +890,16 @@ def example_list_offsets(a, args):
     operation = sys.argv[2]
     args = sys.argv[3:]
 
+    # Custom logger
+    logger = logging.getLogger('AdminClient')
+    logger.setLevel(logging.INFO)
+    handler = logging.StreamHandler()
+    handler.setFormatter(logging.Formatter('%(asctime)-15s %(levelname)-8s %(message)s'))
+    logger.addHandler(handler)
+
     # Create Admin client
-    a = AdminClient({'bootstrap.servers': broker})
+    a = AdminClient({'bootstrap.servers': broker},
```

Comment:
logger should be a property like in the producer and consumer, not an additional parameter
`'logger': logger,`

Reply:
In python, its an additional parameter as well.

Reply:
right, I delete it so I don't request this change